components/automate-chef-io/Makefile

```diff
@@ -53,3 +52,13 @@ lint: themes/chef
 
 build: themes/chef generate_swagger
 	hugo
+
+# Consult scripts/spellcheck.sh in the root Automate directory for notes on using cspell.
+spellcheck:
+	@pushd ../.. > /dev/null && \
+		cspell \
+			components/automate-gateway/**/*.proto \
+			components/automate-chef-io/**/*.md \
+			components/automate-chef-io/layouts/**/*.html \
+			components/automate-chef-io/data/docs/api-static/**/* \
```

Comment:
This appeared to be all the files in this directory that contained spellcheck'able prose that was not a file generated elsewhere in the project.